src/molecules/Address.tsx

```diff
@@ -0,0 +1,21 @@
+import type { FunctionComponent } from 'react';
+
+import type { FlexProps } from '@atoms';
+import { Blockie, Body, Flex } from '@atoms';
+
+interface Props extends Omit<FlexProps, 'variant'> {
+  address: string;
+}
+
+export const Address: FunctionComponent<Props> = ({ address, ...props }) => (
+  <Flex variant="rowAlign" {...props}>
```

Comment:
Since we're starting from scratch it would be a good time to find a correct naming convention for these variants. @guillaume ?

Reply:
I don't have an issue with these names tbh, what is your issue with them?

Reply:
The nature of the alignment is ambiguous at times.
Atm `vertical` and `horizontal` and `center` `start` feel more explicit.

Reply:
Yes it would be more explicit having `vertical-center` or `horizontal-center` for eg.

Reply:
Can you outline what you want them to be named then?

Reply:
- `rowAlign` -> `horizontal-start`
-  `rowCenter` -> `horizontal-center`